README.md

```diff
@@ -71,7 +71,12 @@ wget -O - https://github.com/palomachain/paloma/releases/download/v1.0.0/paloma_
 sudo chmod +x /usr/local/bin/palomad
 
 # Required until we figure out cgo
-sudo wget -P /usr/lib https://github.com/CosmWasm/wasmvm/raw/main/internal/api/libwasmvm.x86_64.so
+systemctl stop palomad
+
+wget https://github.com/CosmWasm/wasmvm/releases/download/v1.2.3/libwasmvm.x86_64.so
+sudo mv libwasmvm.x86_64.so /usr/lib/
+
+systemctl start palomad
```

Comment:
@verabehr let's also remove this line. Not needed as below we give instructions to start paloma.

Reply:
I'll rearrange this section. As far as I understand it, this is really a prerequisite and is not directly part of installing the paloma binary